documentation/release_notes.markdown

```diff
@@ -208,7 +212,7 @@ Justin Holguin, Ken Barber, Kylo Ginsberg, Russell Sim, Ryan Senior and Wyatt Al
     I've also cleaned up a couple of other typos and style items, plus
     removed one more case of < v4 report format checking.
 
-* [PDB-904](https://tickets.puppetlabs.com/browse/PDB-904) Switch fact_values.value_string trgm index to be GIN not GIST 
+* [PDB-904](https://tickets.puppetlabs.com/browse/PDB-904) Switch fact_values.value_string trgm index to be GIN not GIST
```

Comment:
I'd be really wary about doing cleanup patches like this mixed in with something else. This is a 'hot' file now, be mindful.